comp/otelcol/otlp/components/exporter/serializerexporter/consumer.go

```diff
@@ -119,7 +120,7 @@ func (c *serializerConsumer) ConsumeAPMStats(ss *pb.ClientStatsPayload) {
 	c.apmstats = append(c.apmstats, body)
 }
 
-func (c *serializerConsumer) ConsumeSketch(ctx context.Context, dimensions *otlpmetrics.Dimensions, ts uint64, qsketch *quantile.Sketch) {
+func (c *serializerConsumer) ConsumeSketch(ctx context.Context, dimensions *otlpmetrics.Dimensions, ts uint64, interval int64, qsketch *quantile.Sketch) {
```

Comment:
QQ: where is `interval` set and what is the expected value? I'm a bit surprised that you don't need to modify other code base to account for this change